src/main/java/com/exasol/ReportWriter.java

```diff
@@ -0,0 +1,87 @@
+package com.exasol;
+
+import java.io.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import com.exasol.validation.ValidationReport;
+
+/**
+ * This class writes reports from {@link ValidationReport} to a file.
+ */
+public class ReportWriter {
+    private static final Logger LOGGER = Logger.getLogger(ReportWriter.class.getName());
+    private static final String RELEASE_ROBOT_REPORT = "/.release-robot/last_report.txt";
+    private static final String HOME_DIRECTORY = System.getProperty("user.home");
+    private static final String REPORT_PATH = HOME_DIRECTORY + RELEASE_ROBOT_REPORT;
+    private static final String LINE_SEPARATOR = System.getProperty("line.separator");
+    private final UserInput userInput;
+
+    /**
+     * Create a new instance of {@link ReportWriter}.
+     * 
+     * @param userInput instance of {@link UserInput}
+     */
+    public ReportWriter(final UserInput userInput) {
+        this.userInput = userInput;
+    }
+
+    /**
+     * Write a validation report to the file.
+     * 
+     * @param validationReport report to write
+     */
+    // [impl->dsn~rr-writes-validation-report-to-file~1]
+    public void writeValidationReportToFile(final ValidationReport validationReport) {
+        final File reportFile = prepareFile();
+        try (final FileWriter writer = new FileWriter(reportFile, false)) {
+            writeReport(validationReport, writer);
+        } catch (final IOException exception) {
+            throw new IllegalStateException("E-RR-RW-2: Unable to write a report.");
```

Comment:
Add `exception` as cause:
```suggestion
            throw new IllegalStateException("E-RR-RW-2: Unable to write a report.", exception);
```